google-cloud-logging/src/main/java/com/google/cloud/logging/TraceLoggingEnhancer.java

```diff
@@ -19,16 +19,9 @@
 /* Adds tracing support for logging with thread-local trace ID tracking. */
 public class TraceLoggingEnhancer implements LoggingEnhancer {
 
-  private static final String TRACE_ID = "trace_id";
-  private final String traceIdLabel;
+  public TraceLoggingEnhancer() {}
 
-  public TraceLoggingEnhancer() {
-    traceIdLabel = TRACE_ID;
-  }
-
-  public TraceLoggingEnhancer(String prefix) {
-    traceIdLabel = (prefix != null) ? prefix + TRACE_ID : TRACE_ID;
-  }
+  public TraceLoggingEnhancer(String prefix) {}
 
   private static final ThreadLocal<String> traceId = new ThreadLocal<>();
```

Comment:
Are we supposed to use same approach as one used in `ContextHandler` (e.g. use ThreadLocal or InheritableThreadLocal based on config)?

Reply:
I can undo the change since this file is not part of the implementation. The changes here are meant to eliminate compilation warnings only.